Engine/csound_orc_semantics.c

```diff
@@ -299,6 +298,25 @@ CS_VARIABLE* find_var_from_pools(CSOUND* csound, char* varName,
   return var;
 }
 
+/*
+  Check a symbol for pfield format (pN, PN)
+  and return the p-field num ( >= 0 ) 
+  else return -1  
+*/
+static int32_t IsPfield(CSOUND *csound, TYPE_TABLE* typeTable, char *s) 
```

Comment:
Would prefer this to be renamed is_pfield to match function naming style in rest of codebase. In general the pascal/C# conventions of function naming with CapitalCamelCase has started to creep into our codebase and I think it's not good to mix styles. We could have one style designated for the API vs. internal coding, but just need some rule for this to make things clear.

Reply:
That said, we also have camelCase-named functions so there's already an inconsistency; my preference for snake_case is that it seems better in-line for C conventions.

Reply:
I am happy to follow any conventions that are set. Probably a job for a single PR to rename all internal functions according to an agreed style.